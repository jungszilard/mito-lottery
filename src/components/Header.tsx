import Image from "next/image"
import lotteryIcon from "../../public/lottery.svg"

const Header = () => {
  return (
    <div className='h-15 bg-gradien-primary flex items-center'>
      <Image
        priority
        src={lotteryIcon}
        alt='Lottery app icon'
        className='ml-5 mr-8 '
        data-testid='lottery-icon'
      />
      <p className='text-white text-4xl	font-bold capitalize'>
        Lottery simulator
      </p>
    </div>
  )
}

export default Header
