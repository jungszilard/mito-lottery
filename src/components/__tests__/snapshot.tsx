import { render } from "@testing-library/react"
import Header from "../Header"

it("renders homepage unchanged", () => {
  const { container } = render(<Header />)
  expect(container).toMatchSnapshot()
})
