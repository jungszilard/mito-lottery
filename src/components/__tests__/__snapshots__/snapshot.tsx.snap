// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="h-15 bg-gradien-primary flex items-center"
  >
    <img
      alt="Lottery app icon"
      class="ml-5 mr-8 "
      data-nimg="1"
      data-testid="lottery-icon"
      decoding="async"
      fetchpriority="high"
      height="40"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent;"
      width="40"
    />
    <p
      class="text-white text-4xl font-bold capitalize"
    >
      Lottery simulator
    </p>
  </div>
</div>
`;
